src/explore-education-statistics-common/src/components/CollapsibleList.tsx

```diff
@@ -1,48 +1,72 @@
-import React, { useState, Children, ReactNode } from 'react';
+import React, {
+  Children,
+  cloneElement,
+  isValidElement,
+  ReactNode,
+  useState,
+} from 'react';
+import classNames from 'classnames';
 import ButtonText from './ButtonText';
+import styles from './CollapsibleList.module.scss';
 
 interface Props {
   children: ReactNode;
   collapseAfter?: number;
   isCollapsed?: boolean;
+  ordered?: boolean;
+  listStyle?: 'none' | 'number' | 'bullet';
 }
 
 const CollapsibleList = ({
   children,
   collapseAfter = 5,
   isCollapsed = true,
+  ordered = false,
+  listStyle = 'none',
 }: Props) => {
   const [collapsed, setCollapsed] = useState<boolean>(isCollapsed);
 
   if (Children.count(children) > collapseAfter) {
-    if (collapsed) {
-      return (
-        <>
-          {React.Children.map(children, (child, i) => {
-            if (i >= collapseAfter - 2) {
-              return null;
-            }
-            return child;
-          })}
-          {Children.count(children) - (collapseAfter - 2) && (
+    const listItems = Children.map(children, (child, i) => {
+      if (i >= collapseAfter - 2) {
+        if (isValidElement(child)) {
+          return cloneElement(child, {
+            ...child.props,
+            'aria-hidden': false,
+            className: classNames(
+              child.props.className,
+              collapsed && 'govuk-visually-hidden',
+            ),
+          });
+        }
+      }
+      return child;
+    });
+
+    const listClasses = classNames(
+      'govuk-list',
+      styles.listContainer,
+      listStyle === 'number' && 'govuk-list--number',
```

Comment:
Could use object syntax to avoid getting random spaces in the final class string:
```
{
  'govuk-list--number': listStyle === 'number',
  'govuk-list--bullet': listStyle === 'bullet',
}
```